client/layout/masterbar/logged-out.jsx

```diff
@@ -94,14 +94,6 @@ class MasterbarLoggedOut extends React.Component {
 			return null;
 		}
 
-		/**
-		 * Hide signup from from New Site screen. This allows starting with a new Jetpack or
-		 * WordPress.com site.
-		 */
-		if ( startsWith( currentRoute, '/jetpack/new' ) ) {
```

Comment:
`startsWith` might've been initially overdoing this instead of just simple `===` comparison, but let's confirm there aren't any `/jetpack/new/*` routes in Jetpack connect flow. Otherwise we'd affect those too.

Reply:
Would you like me to keep this check, instead of deleting it?

Reply:
Actually looked up in Calypso and realized there are no active `/jetpack/new/*` so this can go. :-) I just remembered seeing something like that but looks like it's only the redirect.

Reply:
Yeah, that's the impression that I had too — no other `jetpack/new/*` routes. I'll keep it deleted then!